test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultDisplayTemplatesTest.cs

```diff
@@ -29,17 +29,17 @@ public class DefaultDisplayTemplatesTest
                     { "<blink>text</blink>", false, "<blink>text</blink>" },
                     { "<blink>text</blink>", true, "&lt;blink&gt;text&lt;/blink&gt;" },
                     { "&'\"", false, "&'\"" },
-                    { "&'\"", true, "&amp;&#39;&quot;" },
```

Comment:
@ajaybhargavb if you're not going to switch from `new HtmlEncoder()` to `new CustomHtmlEncoder()` or `new TestHtmlEncoder()` as part of this issue, you'll need a follow-up issue.  we shouldn't be unit testing @GrabYourPitchforks's code here, despite the specifics of this particular theory data.